packages/android_intent/android/src/main/java/io/flutter/plugins/androidintent/IntentSender.java

```diff
@@ -59,6 +64,30 @@ void send(Intent intent) {
     }
   }
 
+  void showChooser(Intent intent, @NonNull List<String> ignoredPackages) {
+    Intent chooser = null;
+    intent.setPackage(null);
+    intent.setComponent(null);
+
+    // Exclude ignored packages from the displayed list.
+    if (ignoredPackages.size() > 0) {
```

Comment:
Is this if-statement correct? Why do we need to ignore packages to show a chooser?

Reply:
This is a mistake, thanks